travis/script-syntax.py

```diff
@@ -19,14 +16,24 @@ def main():
         if '.git' in parts or 'depends' in parts:
             continue
         for filename in filenames:
-            if not filename.endswith(ext):
+            if not filename.endswith(args.ext):
```

Comment:
I'll change `.travis.yml` then.

The idea was that this makes manual use easier by working even if you specify the `.` - it still works without that change though, false-positives are possible but there are none in the repo.